content/getting-started/registering.md

```diff
@@ -4,8 +4,71 @@ prev: "/getting-started/"
+
+![first task](./img/registering_02_multiply_initial.png)
+
+Task says: "The code does not execute properly. Try to figure out why." Let's take a closer look at it... Of course! Some tiny detail is missing, and it causes compilation error! Let's put it in place, fix the code, and click `SUBMIT`. Server will process our solution and... after a short while, tests report is presented to you. If you made a mistake and your answer is not correct, don't worry! Just click `TRY AGAIN` and fix your solution.
+
+![retry](./img/registering_03_multiply_wrong_answer.png)
+
+
+If your answer is correct, you will experience the best little thing in a life of a coder _[citation needed]_: **green tests**! Congratulations!
+
+![success](./img/registering_04_multiply_success.png)
+
+## Your new account
+
+You have proved yourself worthy. You have proved yourself committed and willing to learn and improve your skills. You are allowed to enter CodeWars dojo. To get your account created, fill in a couple of details, or use your GitHub account to authenticate on CodeWars:
+
+![enlist](./img/registering_05_enlist.png)
+
+Great, your account is created!
+
+**Important note:** CodeWars will send you an email to confirm your email account. Go to your inbox and click on confirmation link in the message titled `"Confirm your account... Dojo Access Link"`, received from CodeWars. Until you do this, you will be allowed to train and solve tasks, but some actions will be restricted or unavaliable for you. For example, you won't be able to create discourse posts. If you cannot find the email, check `Spam` folder of your inbox! If you do not confirm your account right now, you can always ask CodeWars to resend you the confirmation email by clicking dedicated button in your account panel (we will get to your account panel soon).
+
+## Your training routine
+
+There's only one last step you need to do before you start solving tasks: you need to set up your training routine, so CodeWars would know what to suggest you and how to drive you through your learning process. Don't worry, you just need to answer a few simple questions:
+
+![initial training setup](./img/registering_06_initial_training_setup.png)
+
+ - **What is your programming experience?** - select any option which you feel describes you the best. If you are making your first steps in the world of programming, just pick "Learning to Program". If you are a seasoned programmer who wants to train their coding-fu, improve their skills, and probably learn somethng new, you can choose "Senior Developer".
+ - **Would you like to receive our newsletter with weekly challenges?** - choose "Yes" to receive a weekly email with some suggested tasks to solve. Email contains a choice of CodeWars problems of every difficulty, so you will be always able to find something what would suit you. Participating in Weekly Coding Challenge is a great way to keep yourself motivated! If you do not want to receive the email, just pick "No".
```

Comment:
- **Would you like to receive our newsletter with weekly challenges?** - choose "Yes" to receive a weekly email with some suggested tasks to solve. Email contains a choice of Codewars problems of any difficulty, so you will always be able to find something that should suit you. Participating in Weekly Coding Challenge is a great way to keep yourself motivated! If you do not want to receive the email, just pick "No".